test/test_flask_v3.py

```diff
@@ -0,0 +1,109 @@
+import inspect
+from unittest import mock
+
+import pytest
+
+from doctor.flask import handle_http_v3, HTTP400Exception
+from doctor.response import Response
+from doctor.router_v3 import get_params_from_func
+from doctor.types import integer, boolean, Object, new_type
+
+
+ItemId = integer('item id', minimum=1)
+Item = new_type(Object, 'item', properties={'item_id': ItemId})
+IncludeDeleted = boolean('indicates if deleted items should be included.')
+IsDeleted = boolean('Indicates if the item should be marked as deleted')
+
+
+def get_item(item_id: ItemId, include_deleted: IncludeDeleted=False) -> Item:
+    return {'item_id': 1}
+
+
+@pytest.fixture
```

Comment:
This is similar to a setup/teardown function x-unit style test cases.  Anytime this function name is a parameter to a test function it will yield the value to the test then after the test run the remainder of the function which includes the clean up/tear down code.

This will only work in this file, but we can add other `fixtures` to conftest.py if they would be useful to be reused between test files.